docs/en/ingest-management/integrations/uninstall-integration.asciidoc

```diff
@@ -1,12 +1,13 @@
 [[uninstall-integration]]
 = Uninstall an {agent} integration
 
+// REVIEWERS: This topic seems like a duplicate of "Install and uninstall {agent} integration assets"
+// Is there some subtle difference here that I am not understanding? Or maybe
+// there was at some point and that has changed?
```

Comment:
Yeah, I think we could fold the info in this file with the other one after making sure int he info in Step 1 (about removing it from existing policies) is moved. There is no difference otherwise